src/App.tsx

```diff
@@ -1,16 +1,24 @@
 import './App.scss';
+import { useState } from 'react';
 import { MoviesList } from './components/MoviesList';
 import { NewMovie } from './components/NewMovie';
 import moviesFromServer from './api/movies.json';
+import { Movie } from './types/Movie';
 
 export const App = () => {
+  const [movies, setMovies] = useState<Movie[]>(moviesFromServer);
+
+  const addMovie = (movie: Movie) => {
+    setMovies([...movies, movie]);
+  };
```

Comment:
```suggestion
  const handleAddMovie = (movie: Movie) => {
    setMovies(prevMovies => [...prevMovies, movie]);
  };
```
Follow the naming convention. Use an arrow function form of setMovies when operating with the previous value

Reply:
Agreed 👍